index.html

```diff
@@ -1,51 +1,173 @@
 
-  <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
-  <style>
-    .attribution { font-size: 11px; text-align: center; }
-    .attribution a { color: hsl(228, 45%, 44%); }
-  </style>
-</head>
-<body>
+    <title>Frontend Mentor | Notifications page</title>
+  </head>
+  <body>
+    <main class="main">
+      <section id="header" class="section" role="header">
+        <div class="header__container">
+          <h1 class="header__title">
+            Notifications<span class="count">3</span>
+          </h1>
+          <button type="button" class="btn">Mark all as read</button>
+        </div>
+      </section>
+      <section id="notification" class="section">
+        <div class="notification__container">
+          <article class="notification__card notification__card--unread">
+            <figure class="column__left">
+              <img
+                class="notification__avatar"
+                src="./assets/images/avatar-mark-webber.webp"
+                alt="Mark Webber"
+              />
+            </figure>
+            <div class="column__right">
+              <p class="notification__status">
+                <a class="notification__user" href="/">Mark Webber</a>
+                reacted to your recent post
+                <a class="post" href="/">My first tournament today!</a
+                ><span class="dot active"></span>
+              </p>
+              <p class="notification__date">1m ago</p>
+            </div>
+          </article>
+          <article class="notification__card notification__card--unread">
+            <figure class="column__left">
+              <img
+                class="notification__avatar"
+                src="./assets/images/avatar-angela-gray.webp"
+                alt="Angela Gray"
+              />
+            </figure>
+            <div class="column__right">
+              <p class="notification__status">
+                <a class="notification__user" href="/">Angela Gray</a>
+                followed you<span class="dot active"></span>
+              </p>
+              <p class="notification__date">5m ago</p>
+            </div>
+          </article>
+          <article class="notification__card notification__card--unread">
+            <figure class="column__left">
+              <img
+                class="notification__avatar"
+                src="./assets/images/avatar-jacob-thompson.webp"
+                alt="Jacob Thompson"
+              />
+            </figure>
+            <div class="column__right">
+              <p class="notification__status">
+                <a class="notification__user" href="/">Jacob Thompson</a> has
+                joined your group <a class="group" href="/">Chess Club</a
+                ><span class="dot active"></span>
+              </p>
+              <p class="notification__date">1 day ago</p>
+            </div>
+          </article>
+          <article class="notification__card">
```

Comment:
I can see how you appreacite React for looping through components and passing through text.